topics/sequence-analysis/tutorials/ref-based-rad-seq/tutorial.md

```diff
@@ -39,6 +39,10 @@ You can directly used archive from the Sequence Read Archive (SRA) for raw reads
 
 To download all training datasets (i.e reads, reference genome, population map file and barcodes file), you need to use the corresponding [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs) repository.
 
+TODO put on zenodo? get SRA data with SRA tool? 
```

Comment:
Yes to both.

Reply:
I have started uploading fasta reference genome + barcode SRR...10 + detailled population information of SRR....10 on Zenodo specifying the GTN community.... Normaly, I will finish this evening.

Reply:
Zenodo repo created !